app/src/main/res/values/static_strings.xml

```diff
@@ -24,4 +24,11 @@
 
     <!--suppress CheckTagEmptyBody This is a default value for places where we don't want a string set-->
     <string name="empty_string" translatable="false"></string>
+
+
+    <!-- Title for Accessibility Force Enable Zoom Preference -->
+    <string name="preference_accessibility_force_enable_zoom" translatable="false">Zoom on all websites</string>
+    <!-- Summary for Accessibility Force Enable Zoom Preference -->
+    <string name="preference_accessibility_force_enable_zoom_summary" translatable="false">Enable to allow pinch and zoom, even on websites that prevent this gesture.
+</string>
```

Comment:
nit why did the </string> end up on this line? Also are these in static because we want to land this during this sprint without translation?

Reply:
If yes can you also file a followup to move them right after code freeze? 😇